src/components/bottom-bar-item/bottom-bar-item.tsx

```diff
@@ -0,0 +1,73 @@
+import type { AnchorHTMLAttributes, ButtonHTMLAttributes, FC, MouseEventHandler, ReactNode } from 'react'
+import {
+  ElAnchorBottomBarItemContainer,
+  ElBottomBarItemIcon,
+  ElBottomBarItemLabel,
+  ElButtonBottomBarItemContainer,
+} from './styles'
+
+interface CommonBottomBarItemProps {
+  /**
+   * The JSX element to render as the icon
+   */
+  icon: ReactNode
+
+  /**
+   * Defines a string value that labels the current element
+   * While it's improve the accessibility, this will also used as a label part
+   */
+  'aria-label': string
```

Comment:
**issue:** Since we have a visual label, I don't think it's necessary for us to have an `aria-label` as well. The accessible label will implicitly be the text context of the `BottomBarItem`.

**suggestion:** I think we should rename this prop to `label` (or use `children` for the label text) and make it a `ReactNode`. I like that both this prop and `icon` are required though 👍

Reply:
> I think we should rename this prop to label (or use children for the label text) and make it a ReactNode

right `ReactNode` offers more flexibility, as it can accept any valid React component. Passing it as a children prop can improve code structure in certain instances